services/udp/service.go

```diff
@@ -18,10 +18,6 @@ import (
 )
 
 const (
-	// UDPBufferSize is the maximum UDP packet size
-	// see https://en.wikipedia.org/wiki/User_Datagram_Protocol#Packet_structure
-	UDPBufferSize = 65536
```

Comment:
This diff actually removes that const in lieu of a config var. See original commit: 

https://github.com/sebito91/influxdb/commit/3bf5d9c96928d7865d91dbaed54723570941df2e